### PR TITLE
v2.32.0

### PR DESCRIPTION
**Pull request by avinashbo**
- Fix tracing issues specific to BSC

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,5 +5,5 @@ RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v
     mv /app/build/bin/grpc_health_probe-linux-amd64 /app/build/bin/grpc_health_probe && \
     chmod +x /app/build/bin/grpc_health_probe
 
-FROM thorax/erigon:stable
```
Comment (easeev) [inline Dockerfile:8]: Hmmm why did we have `stable` in the master?

**Review by darinvhs — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (avinashbo) [reply Dockerfile:8]: Initial idea was to use to stable to smoothly overcome the Gibbs hard fork issues with BSC. This PR reverts that and the corresponding release/tag has been deleted and we are also not using the stable image with any of the nodes.

Comment (easeev) [reply Dockerfile:8]: I mean you could have a branch with `stable` in it but I see no point having it in `master`

Comment (avinashbo) [reply Dockerfile:8]: True. I was wrong to have it into `master` branch

File: Dockerfile [hunk 2]
```diff
@@ -5,5 +5,5 @@ RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v
...
+FROM thorax/erigon:2.32.0
```
Comment (easeev) [inline Dockerfile:8]: FYI there's `2.32.1` already

Comment (avinashbo) [reply Dockerfile:8]: We do not yet have the corresponding release for `2.32.1`

**Pull request merged**